### ISSUE TITLE
[BUG]Help content is missing from the WCAG20_Table_CapSummRedundant rule - 3813

### ISSUE THREAD

**Issue by marcjohlic** [IBMa/equal-access#597]
Link to internal issue  https://github.ibm.com/IBMa/E2E/issues/3813

rule: WCAG20_Table_CapSummRedundant
test case: test/v2/checker/accessibility/rules/WCAG20_Table_CapSummRedundant_ruleunit/Table-CapAriaDescribedbyBadId.html

The rule and its help content are removed recently in the preview, but when a user selects a different achieve, even the 'Latest Development', the rule shows up without the help content.

![image.png](https://images.zenhubusercontent.com/58d3fa1c51aa2c3057794e7f/15ba1739-9540-4805-90bc-a138711439d9)

![image.png](https://images.zenhubusercontent.com/58d3fa1c51aa2c3057794e7f/1be94c0a-4031-44a4-9148-53fff27f1521)

- [x] `accessibility-checker-engine`
- [x] `accessibility-checker`
- [x] `karma-accessibility-checker`
- [x] `rule-server`

**Comment by shunguoy** [comment]
https://github.com/IBMa/equal-access/pull/611

**Comment by eirckr** [comment]
Validated. 
<img width="1536" alt="Screen Shot 2022-01-14 at 11 12 07 AM" src="https://user-images.githubusercontent.com/62436670/149556602-0f60a298-9e88-45b7-8dfc-39b40c5f8ea9.png">
